scripts/package.sh

```diff
@@ -55,7 +55,7 @@ function main() {
         #use jam
         util::tools::jam::install --directory "${BUILDPACKDIR}/.bin"
         if [[ -z "${version:-}" ]]; then #version not provided, use latest git tag
-            git_tag=$(git describe --abbrev=0 --tags 2>/dev/null || echo "v0.0.0")
+            git_tag=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
Wouldn't this line error out when packaging a new cnb with no tags?

Reply:
yes, but its consistent with all of the other buildpacks